pkg/apis/eksctl.io/v1alpha5/types.go

```diff
@@ -547,6 +548,16 @@ type (
 // NodeGroupKubeletConfig contains extra config parameters for the kubelet.yaml
 type NodeGroupKubeletConfig map[string]interface{}
 
+// DeepCopy is needed to generate kubernetes types for NodeGroupKubeletConfig
+func (in *NodeGroupKubeletConfig) DeepCopy() *NodeGroupKubeletConfig {
```

Comment:
@errordeveloper how do I make sure that by introducing this I didn't break anything? What is the eksctl command to run to exercise this `DeepCopy()` functions?

Reply:
I am not so sure to be honest, I'm surprised this didn't break anything, maybe we need to look into it, but let's  open an issue and review that separately. I'd rather get this PR over the line as it stands first.

Reply:
Maybe it's possible generator behaviour just changed a little. Maybe the runtime used to deep-copy certain things more auto-magically and became more explicit? Fons bumped the version in his PR earlier.